templates/demo.html

```diff
@@ -7,46 +7,75 @@
 {% block content %}
 <script type="module">
   import AnboxStream from '/static/js/anbox-stream-sdk.js';
+  // @ params appName: name of the application to stream
+  function streamContainer(appName, height = false, width = false) {
+    var container = document.getElementById(appName);
+    var errorContainer = container.getElementsByClassName("js-error-container")[0];
```

Comment:
If your using modules why are you not using querySelector?

Reply:
Force of habit... I'll update these.